src/api/VoteMonitor.Api.Location/Services/XslxFileLoader.cs

```diff
@@ -12,10 +13,26 @@ namespace VoteMonitor.Api.Location.Services
     public class XlsxFileLoader : IFileLoader
     {
         private static readonly string[] SUFFIXES = { ".xlsx", ".xls" };
-        private static readonly int COD_JUDET_INDEX = 1;
-        private static readonly int ADRESA_INDEX = 7;
-        private static readonly int SIRUTA_INDEX = 3;
-        private static readonly int NR_SECTIE_INDEX = 5;
+        private static PollingStationExcelHeader header = new PollingStationExcelHeader();
+
+        public byte[] ExportHeaderInformation()
+        {
+            using (var excelPackage = new ExcelPackage())
+            {
+                var workSheet = excelPackage.Workbook.Worksheets.Add("HeaderInfo");
+
+                var maximumColumnIndex = header.Columns.OrderByDescending(x => x.Index).FirstOrDefault().Index;
+
+                List<string> headerColumns = new List<string>();
```

Comment:
Please, use var declaration instead of List<string> to keep the code consistent.

Reply:
Changed it. Thanks